searx/engines/json_engine.py

```diff
@@ -129,7 +130,7 @@ def response(resp):
                 content = ""
             results.append(
                 {
-                    'url': to_string(url),
+                    'url': url_prefix + to_string(url),
```

Comment:
Why is that necessary?

Reply:
because the url returned by the mozilla mdn api is relative, e.g. `/en-US/docs/Web/CSS/:has`. So we need to prepend a base url.